client/my-sites/themes/use-theme-showcase-logged-out-seo-content.js

```diff
@@ -28,7 +28,7 @@ export default function useThemeShowcaseLoggedOutSeoContent( filter, tier ) {
 			recommended: {
 				all: {
 					title: translate( 'WordPress Themes' ),
-					header: translate( 'Find the perfect theme for your website.' ),
+					header: translate( 'Find the perfect theme for your website' ),
```

Comment:
@stephanethomas A heads up that I tagged this PR with `string freeze` as this change will introduce a new phrase to be translated.